### ISSUE TITLE
water testing page

### ISSUE THREAD

**Issue by sunderrajan** [fkanlab/knowledge#1]
This issue documents improvement of the Water testing page https://github.com/fkanlab/knowledge/wiki/Testing-for-Fluoride 

It is based on discussions between Swati and Sunder on Sep 15 2015

Some comments

1. To have technology usage photo for each technology
2. To mention about popularity (in India and abroad) , ease/difficulty of usage for each technology
3.  To mention about capability and limitations of each technology

To find how to insert photos

In Section on Popular use:
1. To know where to go for testing in India - a map of labs would help
2.  Any apps for water testing
3. once this page is done, have a social media campaign around it and embed some of the discussions in this section

**Comment by aadvaark** [comment]
Would also apply for treatment

**Comment by sunderrajan** [comment]
yes Sam, we plan to get out water testing page this week , and then get to water treatment which wld have  similar needs

**Comment by sunderrajan** [comment]
yes Sam, we plan to get out water testing page this week , and then get to water treatment which wld have  similar needs
